Library/Homebrew/nightly-linux/centos7/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM centos:7
+
+# hadolint ignore=DL3008
+RUN yum -y update \
+   && yum groupinstall -y 'Development Tools' \
+   && yum install -y procps-ng curl file
+
+# For git:
+RUN yum install -y zlib-devel
+# For curl:
+RUN yum install -y openssl-devel
+
+RUN curl -sL http://ftp.gnu.org/gnu/tar/tar-1.32.tar.gz | tar xz \
+    && cd /tar-1.32 \
+    && FORCE_UNSAFE_CONFIGURE=1 ./configure --prefix=/usr/local \
+    && make install \
+    && cd / \
+    && rm -rf /tar-1.32
+
+RUN curl -sL https://curl.se/download/curl-7.82.0.tar.bz2 | tar xj \
+    && cd /curl-7.82.0 \
+    && make configure \
+    && ./configure --prefix=/usr/local --with-openssl \
+    && make \
+    && make install \
+    && cd / \
+    && rm -rf /curl-7.82.0
+
+RUN curl -sL http://mirrors.edge.kernel.org/pub/software/scm/git/git-2.28.0.tar.gz | tar xz \
+    && cd /git-2.28.0 \
+    && make configure \
+    && ./configure --prefix=/usr/local --with-curl=/usr/local \
+    && make install NO_TCLTK=1 \
+    && cd / \
+    && rm -rf /git-2.28.0
```

Comment:
I don't think we should be building these from source every night, we should come up with a better solution here for Linux systems.